app/views/content/non-uk-teachers/fees-and-funding-for-non-UK-trainees.md

```diff
@@ -0,0 +1,321 @@
+
+You will generally be eligible for a tuition fee loan and a student maintenance loan if you meet both of the following requirements:
+
+* you are an Irish citizen
+* you have been living in the UK, Guernsey, Jersey or the Isle of Man throughout the 3 years immediately before the start of your course
+
+You will generally be eligible for a tuition fee loan if you meet both the following requirements:
+
+* you are an Irish citizen
+* you have been living in Ireland, or Ireland and the UK, Guernsey, Jersey or the Isle of Man, throughout the 3 years before the start of your course
+
+### Support for trainee teachers  from the European Economic Area (EEA) or Switzerland
+
+If you meet both the following requirements:
+
+* you have settled status under the EU Settlement Scheme
+* you have been resident in the UK throughout the 3 years before the start of your course
+
+you will generally be eligible for a:
+
+* tuition fee loan
+* maintenance loan
+
+If you meet all 3 of the following requirements:
+
+* you have settled or pre-settled status under the EU Settlement Scheme
+* you have been resident in the UK, Gibraltar, the EEA or Switzerland throughout the 3 years before the start of your course
+* you are an EEA or a Swiss worker, or a family member of one
+
+you will generally be eligible for a:
+
+* tuition fee loan
+* maintenance loan
+
+If you meet both the following requirements:
+
+* you are an EU national with settled or pre-settled status under the EU Settlement Scheme
+* you have been resident in the UK, EEA or Switzerland throughout the 3 years before the start of your course
+
+you will generally be eligible for a tuition fee loan.
+
+### Trainee teachers from outside the European Economic Area (EEA) or Switzerland
+
+To be eligible for financial support, you must usually have settled status (for example, indefinite leave to remain) and have been living in the UK, Guernsey, Jersey or the Isle of Man throughout the 3 years immediately before the start of your course. Exceptions apply in some cases – for example if you have been recognised as a refugee.
+
+### Refugees and asylum seekers
+
+If you have been recognised as a refugee by the UK government, you, your spouse or civil partner and your children, are eligible for financial support for teacher training.
+
+You will also need:
+
+* to have been living in the UK since you were recognised as a refugee
+* to be living in England when your teacher training course starts
+
+If you are an asylum seeker you will not be eligible for financial support, but if you are recognised as a refugee after your course has started, you may be able to claim financial support to complete your studies.
+
+Financial support can include:
+
+* help with living expenses (a maintenance loan)
+* help with course fees (a tuition fee loan)
+
+If, as a refugee, you are eligible for help with course fees, you may also be eligible for a bursary or scholarship.
+
+## Types of financial support available
```

Comment:
I wonder if this whole section should come further up the page? Makes more sense to me to explain what these types of support are first before explaining who is eligible for them